e2e/bitrise.yml

```diff
@@ -22,6 +22,7 @@ workflows:
     - BRANCH: test/generate-changelog
     - FETCH_TAGS: "yes"
     - UPDATE_SUBMODULES: "no"
+    - CLONE_DEPTH: 50
```

Comment:
nit: How about using a value other than `50`? Just to test that it indeed applies the custom value and not the constant baked into the step 😄

Reply:
Good call, will do

- [ ] Update `CLONE_DEPTH` in e2e test to verify it's actually working

Reply:
Fixed in https://github.com/bitrise-steplib/steps-git-clone/pull/191/commits/5b99c3e1c3b8311939cd6c06a09c7ad6efd56339 👍